include/llvm/Transforms/IPO/StructFieldCacheAnalysis.h

```diff
@@ -1,18 +1,18 @@
-// llvm/Tranforms/IPO/StructFieldCacheAnalysis.h - Performs Cache-Aware Structure Analysis- C++ -*-===//
```

Comment:
Yes. I add the "*-" because I thought it's useless before but it looks like I should put it there according to LLVM programming guide